### ISSUE TITLE
[Bug] Crash when gesture fast

### ISSUE THREAD

**Issue by laco-dev** [usuiat/Zoomable#221]
Thanks good library.

Sometimes app crashes when gesture fast.
*Image not rendered and size(offset) was unspecified.
I don't know why occurred this situation.

https://github.com/usuiat/Zoomable/assets/28249981/f21ada0e-7572-4ca3-b617-d6a732f24fd8

Library version : `1.6.1`

## Reproduce
- Use snippet below
```kotlin
AsyncImage(
    model = /* network image */,
    contentDescription = null,
    modifier = Modifier
        .fillMaxWidth()
        .zoomable(
            rememberZoomState()
        ),
    contentScale = ContentScale.FillWidth,
)
```
- Do gesture fast (zoom in or zoom out)

**Comment by usuiat** [comment]
@laco-dev 
It looks like the image is moving off screen, not that the app is crashing, is that correct?
If so, you can use `ZoomState.setContentSize()` like the sample code below.
https://github.com/usuiat/Zoomable/blob/8a5caa550a64420f238016edbfca43701197edcc/app/src/main/java/net/engawapg/app/zoomable/BasicSamples.kt#L57

**Comment by laco-dev** [comment]
@usuiat 
Yes, it hasn't crashed yet. but if i gesture on the screen, it will crash.
thanks you for suggestion.

I solved this issue using below. 
```kotlin
AsyncImage(
  modifier = Modifier.onGloballyPositioned {
    if (offset is unspecified) { zoomState.reset() }
  }
)
```
